tests/server/metrics-ga.js: test race fails

Seems the clock can tick over 1 ms, and cause a failure.

```
[ga metrics] describes _calculateQueueTime
  [describes _calculateQueueTime] AssertionError: expected 4001 to equal 4000
    at Test.suite.describes _calculateQueueTime [as test]  <tests/server/metrics-ga.js:95:12>
  [describes _calculateQueueTime] 4000
  [describes _calculateQueueTime] 4001
```

--- a/tests/server/metrics-ga.js
+++ b/tests/server/metrics-ga.js
@@ -87,8 +87,9 @@ suite.tests['describes _calculateQueueTime'] = function () {
   assert.equal(collect._calculateQueueTime(), 0);
   assert.equal(collect._calculateQueueTime(1000), 0);
   assert.equal(collect._calculateQueueTime(1000, 2000, 500), 500);
-  assert.equal(collect._calculateQueueTime(Date.now(), Date.now() + 5000, 1000), 4000);
-  assert.equal(collect._calculateQueueTime(Date.now(), Date.now() - 5000, 1000), 0);
+  const now = Date.now();
+  assert.equal(collect._calculateQueueTime(now, now + 5000, 1000), 4000);
+  assert.equal(collect._calculateQueueTime(now, now - 5000, 1000), 0);
 };
 
 function testMalformedEventData (fixturePath) {

